torchci/components/metrics/panels/ScalarPanel.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * A metrics panel that shows a single scalar value.
+ */
+
+import { RocksetParam } from "lib/rockset";
+import { Box, Paper, Typography, Skeleton } from "@mui/material";
+import useSWR from "swr";
+
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
```

Comment:
we may want to move this fetcher out to somewhere else since it's used like everywhere